src/main/java/org/altbeacon/beacon/BeaconManager.java

```diff
@@ -666,6 +668,36 @@ public static void setDistanceModelUpdateUrl(String url) {
         distanceModelUpdateUrl = url;
     }
 
+    /**
+     * Default class for rssi filter/calculation implementation
+     */
+    protected static Class rssiFilterImplClass = RunningAverageRssiFilter.class;
```

Comment:
Yes, I I just mean use RssiFilter instead of Class like this:

        protected static RssiFilter rssiFilterImplClass = RunningAverageRssiFilter.class;